fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md

```diff
@@ -0,0 +1,159 @@
+
+```bash
+yum install centreon-plugin-Applications-Nginx-Plus-Restapi.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *Nginx Plus* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Nginx Plus :
+
+```bash
+yum install centreon-plugin-Applications-Nginx-Plus-Restapi.noarch
+```
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-applications-nginx-plus-restapi.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Nginx Plus* depuis la page "Configuration > Plugin packs > Manager"
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par environnement Nginx Plus
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle *App-Nginx-Plus-Restapi-custom*. Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory | Name        | Description                                      |
+| :-------- | :---------- | :----------------------------------------------- |
+| X         | APIPORT     | Port used (Default: 443)                         |
+| X         | APIPROTO    | Specify https if needed (Default: 'https')       |
+| X         | APIUSERNAME | Nginx basic username                             |
+| X         | APIPASSWORD | Nginx basic password.                            |
+| X         | APIPATH     | Specify api path (Default: '/api/6')             |
+
+
+## FAQ
+
+### Comment tester un contrôle en ligne de commandes et que signifient les options principales ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commandes depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
+
+```bash
+/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl \
+    --plugin=apps::nginx::nginxplus::restapi::plugin \
+	--mode=connections \
+	--port='443' \
+	--proto='https' \
+	--api-username='myapiuser' \
+	--api-password='myapipassword' \
+	--api-path='/api/6' \
+	--filter-counters='mycountersfilter' \
+	--warning-active='60' \
+	--critical-active='80'
+	--warning-idle='8' \
+	--critical-idle='10' \
+    --warning-accepted='50' 
+	--critical-accepted='65' \
+	--warning-dropped='3' \
+    --critical-dropped='5' \
+	--verbose
+	
+
+OK: Active : 5, Idle : 0, Accepted : 5, Dropped : 0|
+'connections.active.count'=5;;60;80; 'connections.idle.count'=1;;8;10; 'connections.accepted.count'=5;;50;65; 'connections.dropped.count'=0;;3;5;
+```
+
+La commande ci-dessus contrôle les connexions de Nginx Plus (```--mode=connections```).
+Cette commande déclenchera une alarme WARNING si les connexions activent dépasse 60 (```--warning-active='60'```) 
+et une alarme CRITICAL s'il dépasse 80 (```--critical-active='80'```).

```

Comment:
Enhance the explanation about the options and the triggers (or remove some trigger in your example).